src/angular.ts

```
@@ -264,6 +273,176 @@ export abstract class AbstractOidcService<
+                    ))
+            ) {
+                return next(req);
+            }
+
+            if (instance.#isRunningGetParams && instance.#deadlockDetectionTimer === undefined) {
+                instance.#deadlockDetectionTimer = setTimeout(() => {
+                    const name = this.name.replace(/^_/, "");
+
+                    switch (shouldInjectAccessToken) {
+                        case false:
+                            assert(false);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)